frontend/integration-tests/protractor.conf.ts

```diff
@@ -102,6 +102,7 @@ export const config: Config = {
       'tests/login.scenario.ts',
       'tests/base.scenario.ts',
       'tests/secrets.scenario.ts',
+      'tests/add-secrets.scenario.ts',
```

Comment:
Would rather add the TC as part of the `secret.scenario.ts`, since it basically the subset of the secret TC's

Reply:
sounds reasonable, I can add them to `secret.scenario.ts`